mods/d2k/rules/defaults.yaml

```diff
@@ -222,7 +225,7 @@
 	Inherits@2: ^SpriteActor
 	AppearsOnRadar:
 		UseLocation: yes
-	HiddenUnderFog:
+	HiddenUnderShroud:
```

Comment:
IIRC the main reason @OmegaBolt did this was that Carryalls and Ornithopters would otherwise be much harder to shoot down than in the original.